napari/layers/utils/text_manager.py

```diff
@@ -75,7 +75,7 @@ class TextManager(EventedModel):
         The location of the text origin relative to the bounding box.
         Should be 'center', 'upper_left', 'upper_right', 'lower_left', or 'lower_right'.
     translation : np.ndarray
-        Offset from the anchor point.
+        Offset from the anchor point in canvas coordinates.
```

Comment:
It's been a while since I looked at this code, but I _think_`translation`  is in displayed data coordinates.

I think the center point is calculated on the `view_data`, so I think it should be in displayed data coordinates.

https://github.com/napari/napari/blob/6154efe8b30fc418f5ce16ba232d6f2a22f6bbda/napari/layers/utils/_text_utils.py#L31-L49

Reply:
Agreed. @brisvag : I'll let you make this small change and merge this when you're ready.

Reply:
Just to clarify: you mean that just the docstring is wrong?

Reply:
That's what I understood - just a docstring update is needed here.